core/src/main/java/tc/oc/pgm/loot/FillableAndFiller.java

```diff
@@ -0,0 +1,39 @@
+package tc.oc.pgm.loot;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Objects;
+
+public class FillableAndFiller {
+
+  private final LootableMatchModule.Fillable fillable;
+  private final FillerDefinition filler;
```

Comment:
You may be better off using a Pair or immutable pair from guava or apache commons if either of those are available, otherwise it's fine to stay

Reply:
Do spigot/paper also include it? if so, yeah that's fine

Reply:
Not found in paper 1.18. Want me to just make a generic Pair util maybe? Not sure why I just didn't do that